static/sass/partials/_footer.scss

```diff
@@ -0,0 +1,127 @@
+
+.map {
+  padding-left: 0px;
+  padding-right: 0px;
+}
+  .map iframe {
+    padding-right: 0px;
+  }
+.footer-top .row {
+  background: $footer-top-bg;
+  margin-top: 40px;
+  padding-right: 0px;
+}
+.col-12 {
+  padding-right: 0px;
+  padding-left: 0px;
+}
+.col-lg-3 {
+  margin-top: 60px;
+  color: $primary-bg;
+}
+  a {
+    text-decoration: underline;
+    color: $primary-bg;
+  }
+  a:hover {
+    color: $primary-bg;
+  }
+  p {
+    font-size: 12px;
+    font-family: $text-font;
+    text-align: left;
+  }
+  .drupalcebu_header {
```

Comment:
Maybe we should use something more generic? Also this is confusing to work on header for a file called footer. What do you think?